src/OpenFeature.DependencyInjection/MultiTarget/Guard.cs

```diff
@@ -11,4 +11,10 @@ public static void ThrowIfNull(object? argument, [CallerArgumentExpression(nameo
         if (argument is null)
             throw new ArgumentNullException(paramName);
     }
+
```

Comment:
I can't comment on it, but I think this class should be in the DI namespace

Reply:
Done